srcpkgs/backintime/template

```diff
@@ -47,9 +46,5 @@ backintime-qt_package() {
 		vmove usr/share/bash-completion/completions/backintime-qt
 		vmove usr/share/dbus-1
 		vmove usr/share/doc/backintime-qt
-		vmove usr/share/doc/qt
-		vmove usr/share/icons
-		vmove "usr/share/man/man1/backintime-qt*"
```

Comment:
Putting the man page into the same package as the binary seems to make more sense.
The icons and the polkit file also seem to be more likely only used by the qt gui, what is the reason to change this?

Reply:
`xbps-src` complained about `usr/share/doc/qt` not being there in the first place. Let me see where I got it wrong.

Reply:
Yeah, got the error again:
```
=> backintime-qt-1.4.1_1: running pkg_install ...
mv: cannot stat '/destdir//backintime-1.4.1/etc/dbus-1': No such file or directory
=> ERROR: backintime-qt-1.4.1_1: pkg_install: 'mv ${DESTDIR}/$files ${PKGDESTDIR}/${_targetdir}' exited with 1
=> ERROR:   in _vmove() at common/environment/setup/install.sh:228
=> ERROR:   in _noglob_helper() at common/environment/setup/install.sh:12
=> ERROR:   in pkg_install() at srcpkgs/backintime-qt/template:43
```
It tries to move the files out of `backintime` when it builds `backintime-qt`, so I'm confused now.

I reverted the whole `backintime-qt_package` function to the one from master.

Reply:
The reason is that in fact `usr/share/doc` doesn't exist for `backintime-qt`. But as soon as I remove that line, I get:
```
=> backintime-qt-1.4.1_1: running pkg_install ...
mv: cannot stat '/destdir//backintime-1.4.1/etc/dbus-1': No such file or directory
=> ERROR: backintime-qt-1.4.1_1: pkg_install: 'mv ${DESTDIR}/$files ${PKGDESTDIR}/${_targetdir}' exited with 1
=> ERROR:   in _vmove() at common/environment/setup/install.sh:228
=> ERROR:   in _noglob_helper() at common/environment/setup/install.sh:12
=> ERROR:   in pkg_install() at srcpkgs/backintime-qt/template:4
```
where `etc/dbus-1` exists in `backintime-qt` but not in `backintime`.

Reply:
Oh, the `icons` and `polkit` actually do belong to `backintime`, not `backintime-qt`.

Reply:
Judging by the tree output above, I assume the PR changes are correct?

Reply:
The icons and the polkit file are both part of the qt part of it and are in the qt directory in the source. https://github.com/bit-team/backintime/tree/2f7b6099edf40366da1c1020385b300ac2793080/qt

Reply:
Well they are not in the `masterdir` anymore. Not sure how to go about that.

Reply:
I think I got it now. Can you take another look please?

Reply:
i think a lot of the removed `vmove`s need to be restored, see the diff: https://github.com/void-linux/void-packages/actions/runs/7305156960/job/19908370726?pr=47880#step:9:9

Reply:
Is that the diff between the new and previous XBPS package contents?

Reply:
yes

Reply:
Cool, didn't know that we could do that. I'll get back to the PR after the holidays I guess.

Reply:
I still don't understand it... it won't compile until I remove all those `vmmove` lines. Without them, it compiles fine and the Qt GUI also works. The last commit works on my system. Maybe these files just aren't needed anymore by `backintime`? If there's another way, could you please provide a `diff` on `backintime-qt_package()` with your proposal? The one from `master` throws the errors I posted above for all lines that I removed in this PR.

Reply:
the only ones that need to be removed are `vmove etc/dbus-1` and `vmove usr/share/doc/qt`, as they no longer exist in the package.

Reply:
Okay that worked. How did you figure that out if I may ask?

Reply:
> How did you figure that out if I may ask?

I read the error message, checked what was in masterdir/destdir, removed the relevant line, `xbps-src clean`ed, retried

Reply:
Thanks! I'll keep that in mind for the next PRs.

Reply:
Should be good now.